app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -200,7 +200,7 @@ String provideClientId(final @NonNull ApiEndpoint apiEndpoint) {
   @Singleton
   @NonNull HttpLoggingInterceptor provideHttpLoggingInterceptor() {
     final HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
-    interceptor.setLevel(HttpLoggingInterceptor.Level.HEADERS);
+    interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
Oh. I needed this to debug the api. It is only used in debug, but I can revert that.